sourcegraph/sourcegraph
# HK: Remove @types from dependencies and move them to devDependencies

## Description
For some reason we had `@types/*` deps in dependencies list, even though these are dev dependencies.

## Test plan
1. Pull the branch
2. Ensure everything works

## App preview:

- [Web](https://sg-web-og-hk-fix-deps.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-dvyrfjpqop.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
